dev/com.ibm.ws.security.oidc.client_fat/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.rp/configs/rp_server_withOpStub_encrypt.xml

```diff
@@ -0,0 +1,1097 @@
+		trustAliasName="es256"
+		keyManagmetnKeyAlias="rs512"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>	
+	
+	<authFilter id="authFilterES384RS512">
+		<requestUrl
+			id="requestUrlES384RS512"
+			urlPattern="/simple/SignES384EncryptRS512"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignES384EncryptRS512"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterES384RS512"
+		signatureAlgorithm="ES384"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="es384"
+		keyManagmetnKeyAlias="rs512"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>	
+	
+	<authFilter id="authFilterES512RS512">
+		<requestUrl
+			id="requestUrlES512RS512"
+			urlPattern="/simple/SignES512EncryptRS512"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignES512EncryptRS512"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterES512RS512"
+		signatureAlgorithm="ES512"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="es512"
+		keyManagmetnKeyAlias="rs512"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>	
+				
+	<authFilter id="authFilterInvalidKeyManagementKeyAlias">
+		<requestUrl
+			id="requestUrlInvalidKeyManagementKeyAlias"
+			urlPattern="/simple/InvalidKeyManagementKeyAlias"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="invalidKeyManagementKeyAlias"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterInvalidKeyManagementKeyAlias"
+		signatureAlgorithm="RS256"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="rs256"
+		keyManagmetnKeyAlias="ES384"
```

Comment:
What makes this an invalid keyManagementKeyAlias? I'm guessing the JWE would be signed using a different key? But I thought the key aliases in the keystore all use lowercase (e.g. "`rs384"`), in which case this would be pointing to a key that doesn't exist and would be the same as `nonExistantKeyManagementKeyAlias`.

Reply:
It seems to have worked with either case.  We only supported RS algs (build/consumer/mpjwt) - so, didn't know if I'd get a failure because the rp won't allow an es384 cert, or we'd just get a "mismatch" type failure - need to see what the runtime will do.
But, it should not fail because of not finding the cert (case should not be an issue)